charts/akeyless-api-gateway/values.yaml

```diff
@@ -433,6 +433,10 @@ metrics:
 # - logand-conf field (base64). The content the same as above.
 logandExistingSecretName:
 
+# Specifies custom agreement links to appear on login page
+loginPageAgreementLinks:
+#  endUserLicenseAgreement:
```

Comment:
please provide an example for input here with explanation , and a link to akeyless documentation

Reply:
Added example.
Explanation is in line 436.
No documentation yet, as this is a new feature not yet deployed.